scripts/package_sign_tacos.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+# Script for packaging and signing Actian Tableau connectors
+#
+# Parameters passed to script:
+#   1) Build number (required)
+#   2) Jenkins Workspace (required)
+#   3) Branch name (optional)
+#
+# Requirements prior to running script:
+#   - Digicert Software Trust Manager is installed and configured.
+#   - The command "smctl healthcheck" runs successfully, reporting:
+#     - A valid signer certificate is available.
+#     - The user has the privilege to sign.
+#
+
+# Verify that build number was passed in
+if [ -z $1 ]; then
+   echo Missing build number parameter
+   exit 1
+fi
+
+BUILD=$1
+
+# Verify that Jenkins workspace directorywas passed in
+if [ -z $2 ]; then
+   echo Missing workspace directory parameter
+   exit 1
+fi
+
+WKSP=$JENKINS_HOME/$2
+echo Using Jenkins workspace: $WKSP
+
+# Check if branch name was passed
+if [ $3 ]; then
+   BRANCH="-b $3"
+fi
+echo DEBUG BRANCH=$BRANCH
+
+# Check prerequisites
+if [ -z $(which jarsigner) ]; then
+   echo jarsigner not found
+   exit 1
+fi
+
+# Set directory variables
+BASEDIR=$HOME/tableau
+TACODIR=$BASEDIR/tacos
+LOGDIR=$BASEDIR/logs
+SCRIPTDIR=$BASEDIR/scripts
+GITHUBDIR=$BASEDIR/github
+ACTIANDIR=$GITHUBDIR/actian_tableau_connector
+JDBCDIR=$ACTIANDIR/actian_jdbc
+ODBCDIR=$ACTIANDIR/actian_odbc
+TABSDKDIR=$GITHUBDIR/connector-plugin-sdk
+PKGRDIR=$GITHUBDIR/connector-plugin-sdk/connector-packager
+
+# Ensure subdirectories exist
+if [ ! -d "$TACODIR" ]; then
+  mkdir -p $TACODIR/unsigned
+fi
+if [ ! -d "$LOGDIR" ]; then
+  mkdir $LOGDIR
+fi
+
+# Set repo variables
+TABSDKREPO=https://github.com/tableau/connector-plugin-sdk.git
```

Comment:
move to top of script, potentially make this a parameter (optional, with default)

Reply:
Moved variable initialization to top of script (after comments).